lib/Steps/Integrations/NextJs.ts

```diff
@@ -94,8 +94,11 @@ export class NextJs extends BaseIntegration {
     return this.shouldConfigure;
   }
 
-  private _createSentryCliConfig(cliProps: SentryCliProps): void {
-    const authToken = cliProps['auth/token'];
+  private async _createSentryCliConfig(
+    cliProps: SentryCliProps,
+  ): Promise<void> {
+    const cliPropsToWrite = { ...cliProps };
```

Comment:
Let’s instead do
```suggestion
    const { “auth/token”: authToken, ...cliPropsToWrite } = cliProps:
```

that way we don’t gotta use the delete statement.